src/frontend/components/Chat/SharedChatComponents/InputBar/index.tsx

```diff
@@ -16,7 +16,7 @@ const StyledMiniSpinner = styled(Spinner)`
 `;
 
```

Comment:
suggestion commit message :
Once a user has connected to the chat and choosed a display_name and
this ine is available. we save it in the liveRegistration record. Then
when the same user came back again we connect it to the chat with this
same display_name.
=>
Once a user has connected to the chat and **has chosen** a display_name and 
this **one** is available. we save it in the liveRegistration record. Then
when the same user came back again we connect it to the chat with this
same display_name.